compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPULowerToUKernels.cpp

```diff
@@ -397,6 +397,8 @@ static uint32_t flagForUser(IREE::LinalgExt::EncodingUser user) {
     return IREE_UK_FLAG_QUERY_TILE_SIZES_OPERATION_MATMUL_BF16BF16F32;
   case IREE::LinalgExt::EncodingUser::MATMUL_BF16BF16BF16:
     return IREE_UK_FLAG_QUERY_TILE_SIZES_OPERATION_MATMUL_BF16BF16BF16;
+  default:
+    return IREE_UK_FLAG_QUERY_TILE_SIZES_OPERATION_NONE;
```

Comment:
@bjacob what's the most suitable default value for the function to return?

Reply:
bjacob is OOO now. Are we able to use `assert(false)` here?

Reply:
Thanks, will try. 

I actually missed a few more obvious initializations. Will push a new commit.

Reply:
Add both assert and a default value here. GCC warning is not intelligent enough to check the early return.